src/runtime/auto-prompt-manager.js

```diff
@@ -114,25 +133,32 @@ export class AutoPromptManager {
    * configuration, entitlement state, and options specified in params.
    * @param {!../model/client-config.ClientConfig|undefined} clientConfig
    * @param {!../api/entitlements.Entitlements} entitlements
+   * @param {?./entitlements-manager.Article} article
    * @param {{
    *   autoPromptType: (AutoPromptType|undefined),
    *   alwaysShow: (boolean|undefined),
    *   displayLargePromptFn: (function()|undefined),
    * }} params
    * @return {!Promise}
    */
-  showAutoPrompt_(clientConfig, entitlements, params) {
+  showAutoPrompt_(clientConfig, entitlements, article, params) {
     return this.shouldShowAutoPrompt_(
       clientConfig,
       entitlements,
       params.autoPromptType
     ).then((shouldShowAutoPrompt) => {
+      const promptFn =
```

Comment:
Nit - put this block inside the `if (!shouldShowAutoPrompt)` since it's only used if `shouldShowAutoPrompt` is false.

Reply:
Updated to also passing this along to the miniprompt below